### DISCUSSION TITLE
Related to 'Ell'

### DISCUSSION THREAD

**Discussion by DJuego** [taocpp/PEGTL#24] [General]
Hi! Thank you for this contribution. :-) I have a curiosity.

is PEGTL related (historically)  to "Ell - Embedded LL library. A light C++ library to embbed EBFN grammars in your code"?

DJuego

**Comment by ColinH** [comment]
Hi, and thanks for your interest!

Regarding your question, I have to admit that I hadn't even heard of Ell before, so there is no historical connection between the two projects.
